lib/utility/es_client.rb

```diff
@@ -10,26 +10,47 @@
 require 'app/config'
 
 module Utility
```

Comment:
This change makes EsClient an instance class, instead of singleton, should the class be reverted to its original state?

Reply:
Do we really need a singleton here?